### PR TITLE
Removes the chunky fingers check from the modular console.

### PR DESCRIPTION
**Pull request by QuacksQ**
## About The Pull Request
Removes the chunky finger check from the modular console (Basically engineering and others QOL when working with that machine)

Why? Because I hate having to take my gloves off when I'm playing engineering to interact with consoles or to preform other tasks. (Yes this is a salt PR)

## How Does This Help ***Gameplay***?
Engineers can now interact with consoles properly again while they are wearing insuls along with doing other things previously barred because of the chunky fingers trait on these gloves.

## How Does This Help ***Roleplay***?
Minimal impact on Roleplay.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary> 

https://user-images.githubusercontent.com/79924768/232338085-2591067d-4e06-472d-83cd-f95509b5d13f.mp4

</details>

## Changelog
:cl:
del: Removes the chunky fingers trait from Insuls. 
/:cl:

### CODE REVIEW

**Review by RimiNosha — CHANGES_REQUESTED** [review]
Then my recommendation is to remove the chunky fingers block from tablets and computers, unless you want to add a new trait that flags chunky finger gloves that it's given to as usable on them.

**Comment by QuacksQ** [comment]
> Golems and hulks are not staying, that is correct.
> 
> The laser pointer check makes sense, as (especially when upgraded) it can very easily cripple anyone who doesn't have sunglasses (borgs included) if you're an accurate clicker, or they're standing still.
> 
> My recommendation is to remove the chunky fingers block from tablets and computers, unless you want to add a new trait that flags chunky finger gloves that it's given to as usable on them.

Removed the chunky fingers block from computers, Tablets never had them in the first place (and we dont have tablets as of #115)

File: code/modules/clothing/gloves/color.dm [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	dying_key = DYE_REGISTRY_GLOVES
 
 /obj/item/clothing/gloves/color/yellow
-	desc = "These gloves provide protection against electric shock. The thickness of the rubber makes your fingers seem bigger."
+	desc = "These gloves provide protection against electric shock."
```
Comment (RimiNosha) [inline code/modules/clothing/gloves/color.dm:5]: Don't forget to undo the chunky fingers stuff for the gloves and update your PR title and body.

File: code/modules/clothing/gloves/color.dm [hunk 2]
```diff
@@ -12,7 +12,6 @@
 	custom_price = PAYCHECK_CREW * 10
 	custom_premium_price = PAYCHECK_COMMAND * 6
 	cut_type = /obj/item/clothing/gloves/cut
-	clothing_traits = list(TRAIT_CHUNKYFINGERS)
```
Comment (RimiNosha) [inline code/modules/clothing/gloves/color.dm:15]: Ditto.

**Review by RimiNosha — CHANGES_REQUESTED** [review]

**Comment by QuacksQ** [comment]
Requested changes made, PR title and description/comment have been updated. pls re-review for the millionth time 🙏

**Comment by nevimer** [comment]
I am of the opinion that the whole chunky fingers concept is ridiculous anyways. People wear gloves when working with tools and computers, especially in real life, and in game it makes no sense. Trait should be reserved for like, species only, and even then... it doesn't need to exist in our setting?

**Comment by RimiNosha** [comment]
For now, the computers issue is the one that was front and center in this PR, if we wanna have a deep look into how chunky fingers affects things (keeping in mind all races with chunky fingers are gone), then we can make another.

**Review by RimiNosha — APPROVED** [review]
Yeet.

**Pull request merged**